### ISSUE TITLE
Some of the documenation files have a non-free license

### ISSUE THREAD

**Issue by glaubitz** [vlasovskikh/funcparserlib#64]
The doc folder contains multiple files with the license "Creative Commons Attribution-Noncommercial-Share Alike 3.0" which makes those files non-free from a distribution point of perspective and therefore make shipping them with a Linux distribution harder.

Would it be possible to relicense those files or drop them altogether to avoid these licensing issues?

**Comment by vlasovskikh** [comment]
@glaubitz Could you please give some pointers to more information on why the Creative Commons Attribution-Noncommercial-Share Alike 3.0 license is non-free and what problems it causes for a Linux distribution?

**Comment by glaubitz** [comment]
The license excludes commercial use which means it cannot be freely distributed by Linux distributions - which explicitly allow to be used commercially.

From the license text [1]:

```
You may not exercise any of the rights granted to You in Section 3 above in any manner that is primarily intended for or directed toward commercial advantage or private monetary compensation. The exchange of the Work for other copyrighted works by means of digital file-sharing or otherwise shall not be considered to be intended for or directed toward commercial advantage or private monetary compensation, provided there is no payment of any monetary compensation in con-nection with the exchange of copyrighted works.
```

Someone who installs an Ubuntu system which ships the package on their company computer, will violate the license, since using Ubuntu on a company computer qualifies as commercial use.

> [1] https://creativecommons.org/licenses/by-nc-sa/3.0/legalcode

**Comment by vlasovskikh** [comment]
@glaubitz Would it help if I changed the license to [Creative Commons Attribution-ShareAlike 4.0](https://creativecommons.org/licenses/by-sa/4.0/), removing the non-commercial use requirement?

**Comment by glaubitz** [comment]
According to debian-legal, this license is considered to be free: https://wiki.debian.org/DFSGLicenses#Creative_Commons_Attribution_Share-Alike_.28CC-BY-SA.29_v4.0

So, yes. All the licenses mentioned in the wiki as "DFSG-compatible" are considered free by Debian (and consequently all other distributions).

**Comment by glaubitz** [comment]
Any update on this? Would be great if the license could get changed and a new version released so that we can update the package in openSUSE as otherwise the package would have to be moved to the non-free section which I would like to avoid.

Debian currently applies a temporary workaround which repacks the original tarball with those files removed but I don't consider this an ideal solution.

**Comment by vlasovskikh** [comment]
@glaubitz Yes, I've been preparing a new release. I will remove the docs from the distribution and leave them only in the Git repository. I will also replace the license to MIT (the license of the funcparserlib source code), but I guess it won't matter because the docs will no longer be a part of the distribution.

**Comment by vlasovskikh** [comment]
I've published a pre-release version funcparserlib 1.0.0a0 that doesn't include these documentation files. Feel free to try it:

```shell
pip install funcparserlib==1.0.0a0
```
